daemon/graphdriver/driver_linux.go

```diff
@@ -50,7 +50,7 @@ const (
 
 var (
 	// List of drivers that should be used in an order
-	priority = "btrfs,zfs,overlay2,fuse-overlayfs,aufs,overlay,devicemapper,vfs"
+	priority = "overlay2,fuse-overlayfs,btrfs,zfs,aufs,overlay,devicemapper,vfs"
```

Comment:
@AkihiroSuda ptal; I wasn't sure if btrfs/zfs should be tried _before_ or _after_ `fuse-overlays` (I _think_ this is the most logical order, but let me know if not)